src/transformers/models/distilbert/modeling_distilbert.py

```diff
@@ -141,7 +142,10 @@ def __init__(self, config: PretrainedConfig):
         self.dim = config.dim
         self.dropout = nn.Dropout(p=config.attention_dropout)
 
-        assert self.dim % self.n_heads == 0
+        # Have an even number of multi heads that divide the dimensions
+        if self.dim % self.n_heads != 0:
+            # Raise value errors for even multi-head attention nodes
```

Comment:
May I ask you if I can change this comment saying "Raise ValueErrors for uneven multi-head attention nodes"?

Reply:
Sure! Go for it (no need to open a new PR ;) )